nilearn/plotting/img_plotting.py

```diff
@@ -793,7 +793,12 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
         nilearn.plotting.plot_roi : To simply plot max-prob atlases (3D images)
 
     """
-    display = plot_anat(anat_img, cut_coords=cut_coords,
+    if anat_img is not None:
+        bg_img = anat_img
+        warn_str = ("'anat_img' will be removed in future releases. "
+                "Use 'bg_img' instead.")
+        warnings.warn(warn_str)
```

Comment:
We should use here:

warnings.warn(warn_str, DeprecationWarning, stacklevel=2)

See https://docs.python.org/2/library/warnings.html#warnings.warn